data/json/mapgen/shelter.json

```diff
@@ -5,16 +5,29 @@
     "subtype": "collection",
     "//": "now used for shelters that have been looted by kids, see SUS groups for general shelter supplies",
     "items": [
-      [ "jacket_evac", 10 ],
-      [ "emer_blanket", 40 ],
       [ "wrapper", 90 ],
+      [ "evac_pamphlet", 80 ],
       [ "glass_shard", 80 ],
+      [ "emer_blanket", 50 ],
       [ "bottle_plastic", 30 ],
-      [ "plastic_knife", 20 ],
-      [ "plastic_spoon", 20 ],
-      [ "plastic_fork", 20 ],
-      [ "bowl_plastic", 10 ],
-      [ "evac_pamphlet", 80 ],
+      [ "plastic_knife", 25 ],
+      [ "plastic_spoon", 25 ],
+      [ "plastic_fork", 25 ],
+      [ "bowl_plastic", 15 ],
+      [ "jacket_evac", 15 ],
+      [ "folding_poncho", 10 ],
+      [ "sponge", 10 ],
+      [ "mask_dust", 9 ],
+      [ "handflare_dead", 9 ],
+      [ "whistle", 7 ],
+      [ "teddy_bear", 7 ],
+      [ "vinegar", 6 ],
+      [ "candle", 5 ],
+      [ "pliers", 4 ],
+      [ "mirror", 4 ],
+      [ "wearable_light", 2 ],
+      [ "calcium_tablet", 1 ],
+      [ "vitamins", 1 ],
```

Comment:
The shelter_supplies item group is for trash that people didn't take with them because it was too large or not perceived as useful in a FEMA camp, I'm not sure vitamins and head lamps and candles would fall into that; those are more likely to be the exact sort of items people would stuff their pockets with on the way out.

Reply:
I guess I figured they left these items meaning to return, but never did.